preview/HEAD.yml

```diff
@@ -0,0 +1,3 @@
+sources:
+  docs-devex:
+    branches: [DOC-13093-Write-a-tutorial-for-SQL-migration,DOC-13093-Write-a-tutorial-for-SQL-migration-capella]
```

Comment:
Is this necessary if they're about to be merged?

Reply:
No, probably not. 

I couldn't get to build locally, so I had to push the preview change.

I'll remove it.